### DISCUSSION TITLE
ClCapture naming the channels

### DISCUSSION THREAD

**Discussion by DevBruker** [gusmanb/logicanalyzer#47] [Ideas]
Hi,
it would be useful if ClCapture could name the channels when sampling. In this use case ClCapture is started in a loop and exporting the data to a different file for each run. Having the channel names in the captured files would be really helpful.

Regards,

**Comment by gusmanb** [comment]
Sure, I will add it to the next version :)

Can you please open an issue so I can tag it as an enhancement request?

Cheers.
